include/zephyr/net/wifi_mgmt.h

```diff
@@ -1522,6 +1528,50 @@ struct wifi_p2p_params {
 		/** Frequency in MHz (0 = not specified, use default) */
 		unsigned int freq;
 	} connect;
+	/** Group add specific parameters */
+	struct {
+		/** Frequency in MHz (0 = auto) */
+		int freq;
+		/** Persistent group ID (-1 = not persistent) */
+		int persistent;
+		/** Enable HT40 */
+		bool ht40;
+		/** Enable VHT */
+		bool vht;
+		/** Enable HE */
+		bool he;
+		/** Enable EDMG */
+		bool edmg;
+		/** GO BSSID (NULL = auto) */
+		uint8_t go_bssid[WIFI_MAC_ADDR_LEN];
+		/** GO BSSID length */
+		uint8_t go_bssid_length;
+	} group_add;
+	/** Group remove specific parameters */
+	struct {
+		/** Interface name (e.g., "wlan0") */
+		char ifname[CONFIG_NET_INTERFACE_NAME_LEN + 1];
```

Comment:
We should use `NET_IFNAMSIZ` instead of `CONFIG_NET_INTERFACE_NAME_LEN` as the config option might be missing. I suggest we have a separate PR that changes all the use of `CONFIG_NET_INTERFACE_NAME_LEN` in `modules/hostap` as there are many found there. @rado17 or @krish2718 are you able to send a PR for this after this one is merged?